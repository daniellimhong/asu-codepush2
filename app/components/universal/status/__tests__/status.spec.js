import 'react-native';
import React from 'react';
import { Status } from '../';
// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';
import { mount } from 'enzyme';

it('renders correctly', () => {
	 const tree = renderer.create(
			<Status />
   ).toJSON();
   expect(tree).toMatchSnapshot();
});
