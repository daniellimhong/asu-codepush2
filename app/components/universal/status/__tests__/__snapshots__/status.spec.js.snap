// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<RCTScrollView>
  <View>
    <View
      style={
        Array [
          Object {
            "display": "flex",
            "flex": 1,
          },
          undefined,
        ]
      }
    >
      <ActivityIndicator
        animating={true}
        color="black"
        hidesWhenStopped={true}
        size="small"
      />
    </View>
  </View>
</RCTScrollView>
`;
